src/lib/utils.js

```diff
@@ -30,42 +31,81 @@ export const generateReturnUrlToNotesIndex = doc => {
 }
 
 /**
- * Is this permission read only for a note?
- *
- * @param {object} perm - permission node in a io.cozy.permissions document
- * @returns {bool} true if the note should be displayed read-only
+ * Get permissions on io.cozy.files for current context
+ * @private
+ * @param {CozyClient} client
+ * @returns {object}
  */
-function isPermissionReadOnly(perm) {
-  //  PATCH is included by default, and in the ALL shortcut
-  return (
-    perm.verbs &&
-    perm.verbs &&
-    perm.verbs.length > 0 &&
-    !perm.verbs.includes('PATCH') &&
-    !perm.verbs.includes('ALL')
+async function getFilesOwnPermissions(client) {
+  const permissionsData = await client
+    .collection('io.cozy.permissions')
+    .getOwnPermissions()
+  const permissionObject = get(permissionsData, 'data.attributes.permissions')
+  if (!permissionObject) throw `can't get self permissions`
+  const permissions = Object.values(permissionObject)
```

Comment:
to be replaced by `models.permissions.fetchOwn()`?

Reply:
Je me suis posé la question mais ça me gênait d'avoir un getOwnPermissions  dans les collections et un fetchOwn dans les modèles avec un retour sous une autre forme.  Je suis preneur de ton avis

Reply:
le `getOwnPermissions` il est quand même sacrément bas niveau. On utilise jamais le retour tel quel et on est toujours obligé de regarder les attributes et les values. 

Donc avoir une méthode qui retourne les permissions déjà formatées fait complètement sens (et donc le model permission fetchOwn est pas mal en ce sens). 

Par contre, j'aurais mis cette méthode dans la collection directement plutôt que le model pour une meilleure DX et par la suite pouvoir plus facilement gérer du offline ou du cache?